dpti/hti_liq.py

```
@@ -486,21 +486,32 @@ def _post_tasks(iter_name, step, natoms):
 
 def post_tasks(iter_name, natoms):
     fe = einstein.ideal_gas_fe(iter_name)
+    print(f"# fe of ideal gas : {fe:20.12f}")
+
     subtask_name = os.path.join(iter_name, "00.soft_on")
     e0, err0, tinfo0 = _post_tasks(subtask_name, "soft_on", natoms)
```

Ruff (0.15.12)

[warning] Unpacked variable `tinfo0` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)